microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -229,4 +229,18 @@ Adds OLM image references in dedicated file::
 
 Issued: 2024-03-20
 
-{product-title} release 4.15.3 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1257[RHBA-2024:1257] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1255[RHSA-2024:1255] advisory.
\ No newline at end of file
+{product-title} release 4.15.3 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1257[RHBA-2024:1257] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1255[RHSA-2024:1255] advisory.
+
+[id="microshift-4-15-4-dp"]
+=== RHBA-2024:1451 - {microshift-short} 4.15.4 bug fix and enhancement update advisory
+
+Issued: 2024-03-26
+
+{product-title} release 4.15.4 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1451[RHBA-2024:1451] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1449[RHSA-2024:1449] advisory.
+
+[discrete]
+[id="microshift-4-15-4-enhancements"]
+==== Enhancements
+
+Change in openshift-marketplace namespace security::
+* In this release, the `securityContextConfig` parameter is not to be set when creating a catalog source in the openshift-marketplace namespace. However, the `securityContextConfig` parameter must be "restricted" when creating a catalog source in other namespaces. (link:https://issues.redhat.com/browse/OCPBUGS-30034[OCPBUGS-30034])
```

Comment:
QE has confirmed that this is a enhancement. 
CC: @kuiwang02

Reply:
I guess my suggestion then would be to work with QE to write this up so that it sounds more like an enhancement rather than a bug or KI.

Reply:
I suggest 

* With this release, the `securityContextConfig` parameter no longer needs to be changed from the default value of baseline when creating a catalog source in the openshift-marketplace namespace. However, the `securityContextConfig` parameter must be set to restricted when creating a catalog source in other namespaces. (link:https://issues.redhat.com/browse/OCPBUGS-30034[OCPBUGS-30034]) (edited) 

But my question is, does this change impact only the CS used with oc mirror, or all catalog sources we're using with OLM? @pmtk

Reply:
@aravipra 
```console
In this release, you should not set the `securityContextConfig` parameter when creating a catalog source in the openshift-marketplace namespace. However, you must set the `securityContextConfig` parameter to a value of `restricted` when creating a catalog source in other namespaces. (link:https://issues.redhat.com/browse/OCPBUGS-30034[OCPBUGS-30034])
```
from @JoeAldinger is ok. it is same with mine with better word.

@ShaunaDiaz 
`securityContextConfig` has nothing with baseline whether there is this fix or not. baseline work for enforce of namespace which impact catalogsoruce pod creation. but `securityContextConfig`'s value has no baseline.
so your word is not correct.

and more, the fix impact all cs creation what ever it is cs used with oc-miorror or not.

Reply:
@JoeAldinger I do not see the use of  word "you" in the Microshift release notes. Also the word "should" has to be avoided.

Reply:
I agree with @kuiwang02 on
> and more, the fix impact all cs creation what ever it is cs used with oc-miorror or not

For the rest, I'm bit confused - I'm not an expert in that domain so I can only say what I saw and what worked.

When `openshift-marketplace` had security `restricted`: CatalogSource generated by oc-mirror doesn't have `securityContextConfig` so the Pod with the catalog wasn't starting. This could be remedied by addition of `grpcPodConfig.securityContextConfig: restricted` or editing the `openshift-marketplace` ns on the cluster.

After we changed `openshift-marketplace` namespace's security to `baseline`, the CatalogSource from oc-mirror (still without securityContextConfig) worked without modifying it.

I don't know the exact implications and iteration between these two values so I'll let OLM experts decide.

Reply:
done